corehq/apps/app_manager/static_strings.py

```diff
@@ -203,7 +203,7 @@
     gettext_noop('What kind of log transmission the phone should attempt on a daily basis (submitted to PostURL)'),
     gettext_noop('What kind of log transmission the phone should attempt on a weekly basis (submitted to PostURL)'),
     gettext_noop('What the \'Extra Key\' (# on Nokia Phones) does when pressed'),
-    gettext_noop('When a value is selected, this feature controls the display size of any user-provided image such that it will be consistent with the size of the original image file, and consistent across devices. Follow <a href=\'https://confluence.dimagi.com/display/commcarepublic/Image+Sizing+with+Multiple+Android+Device+Models\'> the instructions here</a> to determine which value to use.'),
+    gettext_noop('When a value is selected, this feature controls the display size of any user-provided image such that it will be consistent with the size of the original image file, and consistent across devices. Follow <a href=\'https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143946731/Images+in+CommCare\'> the instructions here</a> to determine which value to use.'),
```

Comment:
This also appears like it could be more specific: https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143946731/Images+in+CommCare#CommCare-Developer-Toolkit-Image-Sizing-with-Multiple-Android-Device-Models